symengine/tests/eval/test_lambda_double.cpp

```diff
@@ -236,10 +236,17 @@ TEST_CASE("Check llvm and lambda are equal", "[llvm_double]")
     r = mul(add(sin(x), add(mul(pow(y, integer(4)), mul(z, integer(2))),
                             pow(sin(x), integer(2)))),
             add(vec));
-    for (int i = 0; i < 4; ++i) {
+    for (int i = 0; i < 1; ++i) {
```

Comment:
Is this intentional to leave this at `1`? What is the purpose of the for loop then?

Reply:
I've changed the tests again.

The expression that was there previously resulted in a huge value and therefore
it was not testing some parts of the expression correctly. Now I'm testing several expressions.